src/org/usfirst/frc/team1492/robot/Winch.java

```diff
@@ -0,0 +1,36 @@
+package org.usfirst.frc.team1492.robot;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.VictorSP;
+
+public class Winch {
+	VictorSP winch;
+	DigitalInput winchKill;
+	
+	enum WinchDirections {OUT, IN, OFF}
```

Comment:
Change the directions to `UP`, `DOWN`, and `STOP`, or something else more appropriate for a winch.